doc/developer/workflow.rst

```diff
@@ -354,15 +354,58 @@ This helps to automatically generate human-readable changelog messages.
 Commit Guidelines
 -----------------
 
-There is a built-in commit linter. Basic rules:
+Commit messages must have a title and optionally a detailed description.
 
-- Commit messages must be prefixed with the name of the changed subsystem, followed
-  by a colon and a space and start with an imperative verb.
+The commit title is a short summary of the commit. adhering to the following format:
 
+- Must be no more than 72 characters
+- Must be a complete sentnce that can be used when generating the release changelog
```

Comment:
maybe the real emphasis should be on a meaningful statement that can be used at release time to help produce a changelog. you could even say "help the release managers out ..." !

Reply:
I added a couple more bullet points to make this clearer.